framework/src/problems/FEProblemBase.C

```diff
@@ -4039,7 +4079,7 @@ FEProblemBase::executeControls(const ExecFlagType & exec_type)
 
     if (!ordered_controls.empty())
     {
-      _control_warehouse.setup(exec_type);
+      _control_warehouse.setupDeprecated(exec_type);
```

Comment:
If we truly want this to be a deprecated function, then we should not call it in framework code

Reply:
Let me give a better name, `combinedSetup`. I did not do consolidation because I think it could be a lot more difficult and we might need to have a design discussion. This part alone can be done quickly.